src/browzine-summon-adapter.js

```diff
@@ -198,34 +198,232 @@ browzine.search = (function() {
   };
 }());
 
+browzine.serSol360Core = (function() {
```

Comment:
Initial goal was to do it in one file, but thinking it might be best to do separate files because looks like there will be a few Discovery Services to tackle in the future. We could have something like this:

Summon:
https://s3.amazonaws.com/browzine-adapters/summon/browzine-summon-adapter.js

360 Core:
https://s3.amazonaws.com/browzine-adapters/360-core/browzine-360-core-adapter.js

I think that should work for customers. Different products, slightly different link. We'll have to tweak the deployment slightly, but should be doable. We'll need to update the README for each product's loader. e.g. Something like:
```
var browzine = {
  api: "https://api.thirdiron.com/public/v1/libraries/XXX",
  apiKey: "ENTER API KEY",
};

browzine.script = document.createElement("script");
browzine.script.src = "https://s3.amazonaws.com/browzine-adapters/360-core/browzine-360-core-adapter.js";
document.head.appendChild(browzine.script);
```

Reply:
Yeah - I'm guessing we can't do Primo in this same code base, correct?

If we could bring that over, well wooooo that would be slick 😄

Reply:
> I think that should work for customers. Different products, slightly different link.

Yeah, I think having slightly different instructions for each one is fine... but I agree, one file to rule them all is pretty sweet 😄

Reply:
> I'm guessing we can't do Primo in this same code base, correct?

Good question. Not sure TBH, but I'll kick the tires in that next Primo JIRA and see what I find.

Reply:
Thanks for checking in that next Primo change, Tian!